src/classes/STG_PanelCreatePayments_CTRL.cls

```diff
@@ -34,26 +34,35 @@
 * @description Controller for the Donor Statistics panel.
 */
 public with sharing class STG_PanelCreatePayments_CTRL extends STG_Panel { 
-
-    
-    /*********************************************************************************************************
+    /**
     * @description Returns the string Id of the Donor Statistics panel. 
     */
     public override string idPanel() { return 'idPanelCreatePayments'; }
     
-    /*********************************************************************************************************
+    /**
+    * @description NPSP namespace: an empty string if unmanaged, or 'npsp' if managed
+    */
+    public String getNamespace() {
```

Comment:
Since all of these controller classes extend from `STG_Panel`, I recommend putting the `getNamespace()` method in there this way it's available to all the classes. I think that'll still work for the getter, but the worst case is that this method in each individual controller simply returns the call from a method in the parent `STG_Panel` class.

Reply:
all `getNamespace()` method are moved to `STG_Panel`